introduction_to_applying_machine_learning/lightgbm_catboost_tabtransformer_autogluon_churn/churn-prediction-lightgbm-catboost-tabtransformer-autogluon.ipynb

```diff
@@ -659,7 +659,7 @@
     "from sagemaker.estimator import Estimator\n",
```

Comment:
<p>Can we revert the number of jobs to the original max_jobs=10? This is a lot of quota being consumed on the CI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3631/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>